qcodes/instrument/parameter.py

```diff
@@ -316,7 +317,7 @@ def __init__(self, name: str,
         self._validate_on_get = False
 
     @abstractmethod
-    def get_raw(self):
+    def get_raw(self) -> Any:
```

Comment:
I added `ParamRawDataType` in https://github.com/QCoDeS/Qcodes/pull/1757 :)
```suggestion
    def get_raw(self) -> ParamRawDataType:
```